src/dtos/monitor-plan-update.dto.ts

```diff
@@ -26,6 +26,8 @@ export class UpdateMonitorPlanDTO {
       return `The value of [${args.value}] for [${args.property}] must be within the range of 1 and 999999.`;
     },
   })
+  version: string;
+
   orisCode: number;
```

Comment:
You've misplaced the decorators intended for `orisCode` here: this will result in improper validation for `version` and no validation for `orisCode`. You should instead assign separate validators to the `version` property.
```suggestion
  })

  orisCode: number;
  
    @ApiProperty({
    description: propertyMetadata.monitorPlanDTOVersion.description,
    example: propertyMetadata.monitorPlanDTOVersion.example,
    name: propertyMetadata.monitorPlanDTOVersion.fieldLabels.value,
  })
  @IsNotEmpty()
  @IsSemVer()
  version: string;
```
This implies that a `monitorPlanDTOVersion` attribute is added to `easey-common`'s `propertyMetadata` object.